frontend/src/metabase/entities/questions.js

```diff
@@ -79,16 +80,34 @@ const Questions = createEntity({
     return state;
   },
 
-  form: {
-    fields: [
-      { name: "name" },
-      { name: "description", type: "text" },
-      {
-        name: "collection_id",
-        title: "Collection",
-        type: "collection",
-      },
-    ],
+  forms: {
+    details: {
+      fields: [
+        { name: "name", title: t`Name` },
```

Comment:
Would it make sense to do something like 

```js
const defaultFields = [
{ name: "name", title: t`Name` },
{
  name: "description",
  title: t`Description`,
  type: "text",
  placeholder: t`It's optional but oh, so helpful`,
}];
```

And then:

```js
forms: {
  details: [
    ...defaultFields,
    { 
       name: "collection_id",
       title: t`Collection`,
       type: "collection",
    }],
    details_without_collection: {
      fields: defaultFields
    }
}
```

just to cut down on the number of places we'd have to change copy were we to. (Also random aside, does ttag dedupe strings automatically?)

Reply:
Yes, ttag dedups